ui/package.json

```diff
@@ -11,13 +11,14 @@
     "test-react": "jest",
     "lint": "ng lint --type-check",
     "e2e": "ng e2e",
-    "codegen": "yarn run merge-swagger && yarn run codegen-clean && yarn run codegen-swagger && yarn run codegen-swagger-fetch && yarn run post-codegen && yarn run notebooks-codegen-swagger && yarn run notebooks-post-codegen",
+    "codegen": "yarn run merge-swagger && yarn run codegen-clean && yarn run codegen-swagger && yarn run codegen-swagger-fetch && yarn run post-codegen && yarn run notebooks-codegen-swagger && yarn run notebooks-codegen-swagger-fetch && yarn run notebooks-post-codegen",
     "merge-swagger": "../api/build.rb merge-yaml --no-docker",
     "codegen-clean": "rm -rf src/generated src/notebooks-generated",
     "codegen-swagger": "java -jar ../aou-utils/swagger-codegen-cli.jar generate --lang typescript-angular --input-spec ../api/src/main/resources/merged.yaml --output src/generated --additional-properties ngVersion=2",
     "codegen-swagger-fetch": "java -jar ../aou-utils/swagger-codegen-cli.jar generate --lang typescript-fetch --input-spec ../api/src/main/resources/merged.yaml --output src/generated/fetch",
     "post-codegen": "sed -i.bak -e 's/OpaqueToken/InjectionToken/' src/generated/variables.ts",
     "notebooks-codegen-swagger": "java -jar ../aou-utils/swagger-codegen-cli.jar generate --lang typescript-angular --input-spec ../api/src/main/resources/notebooks.yaml --output src/notebooks-generated --additional-properties ngVersion=2",
+    "notebooks-codegen-swagger-fetch": "java -jar ../aou-utils/swagger-codegen-cli.jar generate --lang typescript-fetch --input-spec ../api/src/main/resources/notebooks.yaml --output src/notebooks-generated/fetch",
```

Comment:
What does this do?

Reply:
We use swagger to create our API clients but we actually need to generate TWO versions of each since angular and react use different API client libraries.

We were already generating both versions for `codegen` in lines 17-19 but we were only generating the angular version for the notebooks client since none of the react components hit the notebooks API until this PR.

This line adds the step to generate the React version of the notebooks API.